portfolio/src/main/webapp/index.html

```diff
@@ -2,35 +2,62 @@
     <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
-        
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.0-beta1/dist/css/bootstrap.min.css" rel="stylesheet">
+    
+    
+    <link href="style.css" rel="stylesheet" type="text/css" />
     <script src="script.js"></script>
   </head>
   <body>
-    <div id="content">
-      <h1>Tasha Ogoti's Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <div>
-        <img src="/images/blkwoman.jpg"/>
-        <p >My name is Tasha Ogoti.
-            <br> A junior student studying Computer Science at the University of Missouri - Columbia.
-            <br> Born and raised in Nairobi, Kenya.
-        </p>
-
+    <div  class="mainIntro">
+      <div class="navbar">
+        <img src="images/logo.png" class="logo" alt="myLogo" onclick = "window.location.assign(contactList.html)">
+        <nav>
+          <ul id="menu">
+            <li><a href="index.html" id="home">Home</a></li>
+            <li><a href="about.html" id="about">About</a></li>
+            <li><a href="projects.html" id="project">Projects</a></li>
+            <li><a href="contact.html" id="contact">Contact</a></li>
+          </ul>
+        </nav>
+        <img src="images/menuLogo.png" class="menuLogo" onclick="toggleMenu()" alt="bars">
       </div>
 
-      <div id="profiles">
-        <a href="https://www.linkedin.com/in/tasha-ogoti" class="fa fa-linkedin"></a>
-        <a href="https://www.github.com/s3nnah39" class="fa fa-github"></a>
-      </div>
+      <div class="myInfo">
+        <div class="col-1">
+          <h2 class="Ty">Tasha Ogoti<br> Computer Science Student</h2>
+          <h3 class="Ty">Prospective Software Engineer</h3>
+          <p class="Ty">2023 Graduate</p>
+          <a class="resume Ty" href="Ogoti_Tasha.pdf" download="Tasha_Ogoti_2023">Resume</a>
+        </div>
+        <div class="col-2">
+          <img src="images/me.png" class="you" alt="girl">
+          <div ></div>
+        </div>
+        <h3 id="comment"></h3>
+        <div class="likes">
+          <i class="fa fa-thumbs-o-up" style="font-size:36px" id="thumb" onclick="getLikes()"></i>
+        </div>
+    </div>
+  </div>
 
+  <div class="connectMe">
+      <img src="images/github.png" onclick="window.location.assign('https://www.github.com/S3nnah39')" alt="githubLogo">
+
+      <img src="images/linkedin.png" onclick="window.location.assign('https://www.linkedin.com/in/tasha-ogoti')" alt="linkedinLogo">
+  </div>
+
+<!--
+    <div id="content">
       <p>Click <a href="https://mapmonkey.s3nnah39.repl.co/index.html" target="_blank">here</a> to see content a website I have ever created.</p>
       <p>Click here to get a random message:</p>
       <button onclick="addRandomGreeting()" id="helloBtn">Hello</button>
       <div id="greeting-container"></div>
     </div>
+-->
```

Comment:
If this isn't needed anymore, let's remove it so that our files aren't cluttered with unused code.